wcfsetup/install/files/lib/system/template/TemplateScriptingCompiler.class.php

```diff
@@ -38,6 +38,69 @@ class TemplateScriptingCompiler {
 		'eval', 'virtual', 'call_user_func_array', 'call_user_func', 'assert' // code execution
 	];
 	
+	/**
+	 * PHP functions and modifiers that can be used in enterprise mode
+	 * @var	string[]
+	 */
+	protected $enterpriseFunctions = [
+		'array_keys',
+		'array_pop',
+		'array_slice',
+		'array_values',
+		'ceil',
+		'concat',
+		'constant',
+		'count',
+		'currency',
+		'date',
+		'defined',
+		'empty',
+		'end',
+		'explode',
+		'floor',
+		'function_exists',
+		'gmdate',
+		'hash',
+		'implode',
+		'in_array',
+		'is_array',
+		'intval',
+		'is_subclass_of',
+		'isset',
+		'key',
+		'lcfirst',
+		'mb_strpos',
+		'mb_strlen',
+		'mb_strpos',
+		'mb_strtolower',
+		'mb_strtoupper',
+		'mb_substr',
+		'md5',
+		'microtime',
+		'min',
+		'preg_match',
+		'preg_replace',
+		'print_r',
+		'rawurlencode',
+		'reset',
+		'round',
+		'sha1',
+		'spl_object_hash',
+		'strpos',
+		'strlen',
+		'strtolower',
+		'strtotime',
+		'strtoupper',
+		'str_repeat',
+		'str_replace',
+		'str_ireplace',
+		'substr',
+		'trim',
+		'ucfirst',
+		'urlencode',
+		'wcfDebug'
```

Comment:
`base64_encode` is used in some plugins, maybe add the function to the list, too.

Reply:
`max` is also missing, but `min` is supported.